src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Linux/LinuxUtilizationParserCgroupV2.cs

```diff
@@ -545,26 +548,33 @@ private static bool TryGetCgroupRequestCpu(IFileSystem fileSystem, out float cpu
 
         if (cpuPodWeightBuffer.IsEmpty || (cpuPodWeightBuffer.Length == 2 && cpuPodWeightBuffer[0] == '-' && cpuPodWeightBuffer[1] == '1'))
         {
-            Throw.InvalidOperationException($"Could not parse '{_cpuPodWeight}' content. Expected to find CPU weight but got '{new string(cpuPodWeightBuffer)}' instead.");
+            Throw.InvalidOperationException(
+                $"Could not parse '{_cpuPodWeight}' content. Expected to find CPU weight but got '{new string(cpuPodWeightBuffer)}' instead.");
         }
 
         _ = GetNextNumber(cpuPodWeightBuffer, out long cpuPodWeight);
 
         if (cpuPodWeight == -1)
         {
-            Throw.InvalidOperationException($"Could not parse '{_cpuPodWeight}'. Expected to get an integer but got: '{cpuPodWeight}'.");
+            Throw.InvalidOperationException(
+                $"Could not parse '{_cpuPodWeight}' content. Expected to get an integer but got: '{cpuPodWeight}'.");
```

Comment:
Shouldn't this include the buffer? The value here will always be -1.

Reply:
Thank you @danmoseley, I totally missed this one.
This, however, reinforces the need to have tests that validate exception messages (see https://github.com/dotnet/extensions/pull/5324#discussion_r1701099978).

@evgenyfedorov2 if you could add tests - that'd be great. The best way to have such tests in my experience is to use Verify to assert blobs (e.g., text). This way we don't have to maintain/update test assertions manually whenever text changes.

Reply:
Done

Reply:
Awesome, thank you